### PR TITLE
Remove trailing slash from base

### PR DESCRIPTION
**Pull request by cowboyd**
## Motivation 
The installer URL `deno install -A https://frontside.com/platformscript//pls` has one two many slashes

## Approach

Remove one

## Screenshots

image

### CODE REVIEW

**Review by taras — APPROVED** [review]
👍

File: www/routes/docs/[...slug].tsx [hunk 1]
```diff
@@ -50,7 +50,10 @@ export default createMainPage(function DocsPage(props: PageProps<Data>) {
   if (props.data.page.data.description) {
     description = String(props.data.page.data.description);
   }
-  let installer = props.data.base !== '/' ? `${props.data.base}/pls` : `${props.url.protocol}//${props.url.host}/pls`
+  let base = props.data.base?.trim().replace(/\/$/, "") ?? "";
```
Comment (wKich) [inline www/routes/docs/[...slug].tsx:53]: What about this?
```suggestion
  let base = props.data.base ? new URL(props.data.base).origin : "";
```

**Review by wKich — APPROVED** [review]
Looks good

File: www/routes/docs/[...slug].tsx (hunk 1 above)
Comment (cowboyd) [reply www/routes/docs/[...slug].tsx:53]: The base url could have a path component to it:

```ts
new URL('https://frontside.com/platformscript').origin
// https://frontside.com
```

**Pull request merged**